deployment/src/kubernetes/DigitalOcean.md

```diff
@@ -1,84 +0,0 @@
-# DigitalOcean
```

Comment:
I haven't seen that you moved this to somewhere …

Are you sure that we shall remove all the docu?